main.go

```diff
@@ -128,6 +132,11 @@ func main() {
 		SyncPeriod: custodianSyncPeriod,
 	})
 
+	if err != nil {
+		setupLog.Error(err, "Unable to initialize etcd custodian controller with image vector")
+		os.Exit(1)
+	}
```

Comment:
This error checking here seems to be misplaced - `err` is not being assigned anything since the last check.